README.md

```diff
@@ -70,18 +70,85 @@ DeclareSchema::Migration::Migrator.before_generating_migration do
 end
 ```
 
-## Declaring Character Set and Collation
-_Note: This feature currently only works for MySQL database configurations._
+### Global Configuration
+Configurations can be set at the global level to customize default declaration for the following values:
 
-MySQL originally supported UTF-8 in the range of 1-3 bytes (`mb3` or "multi-byte 3")
-which covered the full set of Unicode code points at the time: U+0000 - U+FFFF.
-But later, Unicode was extended beyond U+FFFF to make room for emojis, and with that
-UTF-8 require 1-4 bytes (`mb4` or "multi-byte 4"). With this addition, there has
-come a need to dynamically define the character set and collation for individual
-tables and columns in the database. With `declare_schema` this can be configured
-at three separate levels
+#### Text Limit
+The default text limit can be set using the `Generators::DeclareSchema::Migration::Migrator.default_text_limit=` method.
+Note that a `nil` default means that there is no default-- so every declaration must be explicit.
+This will `raise` a `limit: must be provided for field :text...` error when the default value is `nil` and there is no explicit
+declaration.
 
-### Global Configuration
+For example, adding the following to your `config/initializers` directory will
+set the default `text limit` value to `0xffff_ffff`:
```

Comment:
Similar to comments below, can you make this example something other than the default default? For example, `0xffff` would be good.